extensions/stackdriver/log/logger.cc

```diff
@@ -50,13 +50,17 @@ using google::protobuf::util::TimeUtil;
 
 // Name of the HTTP server access log.
 constexpr char kServerAccessLogName[] = "server-accesslog-stackdriver";
+constexpr char kAuditAccessLogName[] = "server-accesslog-stackdriver-audit";
 
 Logger::Logger(const ::Wasm::Common::FlatNode& local_node_info,
                std::unique_ptr<Exporter> exporter, int log_request_size_limit) {
   // Initalize the current WriteLogEntriesRequest.
   log_entries_request_ =
       std::make_unique<google::logging::v2::WriteLogEntriesRequest>();
 
+  audit_entries_request_ =
```

Comment:
check this out... I have made it a map: https://github.com/istio/proxy/pull/2955/files#diff-d116ca95f1292dece4d352210d220a13 so would be easier to add many types of entries now...